src/stylus/components/_data-table.styl

```
@@ -33,25 +33,25 @@ theme(v-datatable, "v-datatable")
       cursor: pointer
       outline: 0
 
-      i
+      i, svg
```

```suggestion
      .v-icon
```

You also missed one on line 17